managers/restore-manager/RestoreTaskPoller.js

```diff
@@ -29,30 +30,33 @@ class RestoreTaskPoller {
       return RestoreService.createService(plan)
         .then(restoreService => restoreService
           .getRestoreOperationState(restore_opts)
-          .catch(e => logger.error('Caught error in poller', e))
           .then(operationStatusResponse => {
             logger.info(`Got restore operation response for guid ${changedOptions.restore_guid}`, operationStatusResponse);
-            if (_.includes([CONST.APISERVER.RESOURCE_STATE.SUCCEEDED, CONST.APISERVER.RESOURCE_STATE.FAILED], operationStatusResponse.state)) {
-              // cancel the poller and clear the array
-              clearInterval(interval);
-              RestoreTaskPoller.pollers[object.metadata.name] = false;
-            }
             return Promise
-              .try(() => {
-                if (utils.isServiceFabrikOperationFinished(operationStatusResponse.state)) {
-                  return this._logEvent(restore_opts, CONST.OPERATION_TYPE.RESTORE, operationStatusResponse);
-                }
-              })
-              .then(() => eventmesh.apiServerClient.patchResource({
+              .try(() => eventmesh.apiServerClient.patchResource({
                 resourceGroup: CONST.APISERVER.RESOURCE_GROUPS.RESTORE,
                 resourceType: CONST.APISERVER.RESOURCE_TYPES.DEFAULT_RESTORE,
                 resourceId: changedOptions.restore_guid,
                 status: {
                   'state': operationStatusResponse.state,
                   response: operationStatusResponse
                 }
-              }));
+              }))
+              .then(() => {
+                if (_.includes([CONST.APISERVER.RESOURCE_STATE.SUCCEEDED, CONST.APISERVER.RESOURCE_STATE.ABORTED], operationStatusResponse.state)) {
+                  logger.debug('Cancel the poller:', object.metadata.name);
```

Comment:
Please include `Clearing Restore Task Poller`, `Starting Restore Task Poller` etc in logger statements

Reply:
Updated as `Clearing Restore Task Poller`
`Starting poller for restore` is already at [line 66](
https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/381/files/c55a27eb80f29ae1b8ac009adf04785485c81e42#diff-8c64b223c0a29cc710c7f01ce1d89d8dR66)